forge-gui/res/cardsfolder/v/venture_forth.txt

```diff
@@ -2,6 +2,8 @@ Name:Venture Forth
 ManaCost:3 G
 Types:Sorcery
 K:Suspend:3:1 G
-A:SP$ DigUntil | Defined$ You | Valid$ Permanent.Land | ValidDescription$ land | FoundDestination$ Battlefield | RevealedDestination$ Exile | SubAbility$ DBChange | SpellDescription$ Exile cards from the top of your library until you exile a land card. Put that card onto the battlefield and the rest on the bottom of your library in a random order. 
-SVar:DBChange:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | WithCountersType$ TIME | WithCountersAmount$ 3 | SpellDescription$ Exile CARDNAME with three time counters on it.
+A:SP$ DigUntil | Defined$ You | Valid$ Permanent.Land | ValidDescription$ land | FoundDestination$ Battlefield | RevealedDestination$ Exile | RememberRevealed$ True | SubAbility$ DBChange | SpellDescription$ Exile cards from the top of your library until you exile a land card. Put that card onto the battlefield and the rest on the bottom of your library in a random order. 
```

Comment:
there is something wrong with this, shouldn't the Rest to the Bottom be done before this Spell is exiled with Counters?